fsw/src/lc_tbl.h

```diff
@@ -90,30 +90,30 @@ typedef struct
  */
 typedef struct
 {
-    uint8 DefaultState;       /**< \brief Default state for this AP (enumerated)
-                                          States are defined in lc_msgdefs.h        */
-    uint8 MaxPassiveEvents;   /**< \brief Max number of events before filter
-                                          - RTS not started because AP is passive   */
-    uint8 MaxPassFailEvents;  /**< \brief Max number of events before filter
-                                          - AP result transition from pass to fail  */
-    uint8 MaxFailPassEvents;  /**< \brief Max number of events before filter
-                                          - AP result transition from fail to pass  */
-    uint16 RTSId;             /**< \brief RTS to request if this AP fails           */
-    uint16 MaxFailsBeforeRTS; /**< \brief How may consecutive failures before
-                                          an RTS request is issued                  */
-
-    uint16 RPNEquation[LC_MAX_RPN_EQU_SIZE]; /**< \brief Reverse Polish Equation that
-                                                         specifies when this actionpoint
-                                                         should fail                     */
+    LC_APState_t DefaultState; /**< \brief Default state for this AP (enumerated)
```

Comment:
this should stay `uint8` because it is a table (enums are not defined length)

Reply:
Updated.
Forgot to consider that aspect.